Makefile

```diff
@@ -1,22 +1,42 @@
 # Current Operator version
 VERSION ?= 0.0.1
 
+# Current bundle version
+BUNDLE_VERSION ?= 1.0.0
```

Comment:
SDK scaffolding [recycles `VERSION` for this][1].  I don't think we need a separate variable.  It's going to be set at build-time anyway, and we can always set it differently when building the bundle and operator if we feel a need to have divergent values.

[1]: https://github.com/openshift/cincinnati-operator/blob/0731f08b2fbd3989b4642ff55c6a216e97d4c990/Makefile#L4

Reply:
It's defined as "semantic version of the operator in the generated bundle". How ever our operator version is a rather ugly git generated version. And as I recall this shows up in the hub as part of the meta data. I don't think we should use VERSION if the intent is to have it have different value for operator vs. the bundle.

Reply:
I am not sure if the internal binary version shows up in the hub. Are you sure?

Reply:
I'm referring to the version that gets passed in below which is the bundle version which is defined as "semantic version of the operator in the generated bundle":

`operator-sdk generate bundle -q --overwrite --version $(BUNDLE_VERSION)`

And ends up here:
https://github.com/openshift/cincinnati-operator/blob/cde0dc77e7fadd5425e7be7e1b5c0a663e8bc423/config/manifests/bases/update-service-operator.clusterserviceversion.yaml#L102

Reply:
I see, that's makes sense. I think we should reuse `version` for bundle version. During build time we have to make sure use a meaningful version. Which is inline to what we have discussed till now.  Also we can change from commit id to something else in future as well.

Reply:
+1 to one `VERSION` and doing whatever we want to set it to a meaningful value at build time.  The commit ID is just a default for when folks don't care enough to set a meaningful version.

Reply:
done